android/common/src/main/java/com/revenuecat/purchases/common/mappers.kt

```diff
@@ -258,6 +254,15 @@ fun List<*>.convertToJsonArray(): JSONArray {
     return writableArray
 }
 
+fun JSONObject.convertToMap(): Map<String, String?> =
+    this.keys().asSequence<String>().associate { key ->
+        if (this.isNull(key)) {
```

Comment:
will this map null to null? would we want to skip instead?

Reply:
It maps `JSON.Null` to `null`, and we want the nulls in the map